java-api.md

```diff
@@ -52,7 +52,7 @@ that accepts the `AnnotatedText` object.
 ## Multi-Threading
 
 The `JLanguageTool` class is not thread safe. Create one instance of `JLanguageTool`
-per thread, but create the language only once (e.g. `new BritishEnglish()`) and
+per thread, but create the language only once (e.g. `Languages.getLanguageByName("English (GB)")`) and
```

Comment:
Could you suggest using `getLanguageForShortCode("en-GB")` here instead? Using the code seems more robust than using the name. (Same for the other occurrences of `getLanguageForName`)

Reply:
Thank you for the suggestion. Because I don't know a preference of "By*" methods, it is welcome to change.